.github/workflows/build.yml

```diff
@@ -112,8 +114,8 @@ jobs:
       - name: cargo test (debug; rustls-provider-test; all features)
         run: cargo test --all-features -p rustls-provider-test
 
-      - name: cargo package --all-features -p ${{ env.MAIN_CRATE }}
-        run: cargo package --all-features -p ${{ env.MAIN_CRATE }}
+      # TODO ADD NICE name for this step
+      - run: cargo package ${{ env.ALL_FEATURES_EXCEPT_UNSAFE }} -p ${{ env.MAIN_CRATE }}
```

Comment:
minor nit: I think it would be ideal to raise & reference an issue to track this TODO item.

But OTOH I personally have a little doubt about adding separate step labels in general. I think it would be ideal if in general each `run:` command would be both concise & self-descriptive enough to avoid the need for separate step labels. For example, an admin script could be added for some more complex cargo commands, as seems to be already the case in some places.